language.md

```diff
@@ -60,4 +60,7 @@ Ja runā latviski, [dodies šeit](./latvian/zefirs.md).
 
 Hvis du taler dansk, [tryk her](danish/velkommen.md)
 
-Ako govoris hrvatski, [klikni ovdje](croatian/sljez.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.